deploy/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
       - AUTH_ISSUER
     command:
       - --db-connection-string=postgresql://postgres:postgres@db:5432
-      - --use-auth=${USE_AUTH:-false}
+      - --auth-provider=${AUTH_PROVIDER:-none}
```

Comment:
We should revisit the integration tests as I believe this was being used there to run the auth-flow always. We should do that in a separate PR.